src/harness/testcases/heartbeat_testcase.py

```diff
@@ -742,6 +744,68 @@ def test_WINNF_FT_S_HBT_15(self):
         datetime.strptime(response['transmitExpireTime'], '%Y-%m-%dT%H:%M:%SZ'),
         transmit_expire_time_1)
 
+  def heartbeatRequest(self, cbsd_id, grant_id):
```

Comment:
Let's not create helper functions, especially when they have generic names but test-specific behavior (e.g. acceptable response codes). I don't think you gain anything from this helper function, especially if you remove the use of Timer() as suggested below.